Guess game/Program.cs

```diff
@@ -10,282 +10,98 @@ namespace Guess_game
 {
     class Program
     {
+        private static void AnswerMessage(bool correct)
+        {
+            Console.WriteLine(correct ? "Nice you get it " : "You did not get it ");
+
+            Console.Write("Press Enter to continue ");
+            Console.ReadLine();
+        }
+
+
         private static Dictionary<string, string> Riddles = new Dictionary<string, string>
         {
-            // { key, value }
-            {"zalia zole bet ne zole su uodega bet nepele?" , "agurkas"},
-            { "be ranku be koju duris atidaro", "vejas"},
+            {"zalia zole bet ne zole su uodega bet ne pele?" , "agurkas"},
+            { "be ranku be koju duris atidaro?", "vejas"},
             { "I pakalne letai i kalna greitai?", "snarglys"}
         };
-        // foreach 
 
         static void Main(string[] args)
         {
-            //var answers = new Dictionary<string, int>();
-            
+            int Result = 0;
```

Comment:
variables must be named in `camelCase` pattern (first letter is lower cased, you can read about camel case in google)